pkg/controller/resource/deploymentconfig.go

```diff
@@ -0,0 +1,70 @@
+package resource
+
+import (
+	"errors"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	chaosTypes "github.com/litmuschaos/chaos-operator/pkg/controller/types"
+)
+
+var (
+	gvr = schema.GroupVersionResource{
+		Group:    "apps.openshift.io",
+		Version:  "v1",
+		Resource: "deploymentconfigs",
+	}
+)
+// CheckDeploymentAnnotation will check the annotation of deployment
+func CheckDeploymentConfigAnnotation(clientSet dynamic.Interface, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, error) {
```

Comment:
Can we use this variable `dynamicClient` instead to `clientSet`

Reply:
since, In future we are migrating to dynamic-client, its good to have this name

Reply:
Still we will use `dynamicClient` and remove `clientSet`

Reply:
`dynamicClient` set is very different from normal `clientSet` thats why it would be better to let anyone going through the code know that this function is using `dynamicClient`